src/services/ExposureNotificationService/ExposureNotificationService.ts

```diff
@@ -180,6 +180,23 @@ export class ExposureNotificationService {
       });
   }
 
+  public isReminderNeeded(exposureStatus: ExposureStatus) {
+    if (exposureStatus.type !== ExposureStatusType.Diagnosed) {
+      return false;
+    }
+
+    if (exposureStatus.needsSubmission) {
+      return true;
+    }
```

Comment:
Comparing this to the old logic, I think it should be:
```
if (!exposureStatus.needsSubmission) {
  return false;
}
```

Reply:
```
if (
      exposureStatus.type === ExposureStatusType.Diagnosed &&
      exposureStatus.needsSubmission &&
      (!exposureStatus.uploadReminderLastSentAt ||
        minutesBetween(new Date(exposureStatus.uploadReminderLastSentAt), new Date()) >
          MINIMUM_REMINDER_INTERVAL_MINUTES)
    ) {
```